### PR TITLE
iMac--product-card

### PR DESCRIPTION
**Pull request by Mykola-Hadupiak**
[DEMO LINK](https://Mykola-Hadupiak.github.io/layout_product-cards/)
[TEST REPORT LINK](https://Mykola-Hadupiak.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
+        alt="iMac"
+        class="card__image"
+      >
+
+      <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__stars">
+        <div class="stars card__stars-stars--4">
```
Comment (SanyaBratashchuk) [inline src/index.html:24]:
```suggestion
        <div class="stars stars--4">
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
add all files

File: src/index.html (hunk 1 above)
Comment (Mykola-Hadupiak) [reply src/index.html:24]: how i can do this?

File: src/styles/star.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__stars-stars--1 :nth-child(-n + 1),
+.card__stars-stars--2 :nth-child(-n + 2),
+.card__stars-stars--3 :nth-child(-n + 3),
+.card__stars-stars--4 :nth-child(-n + 4),
+.card__stars-stars--5 :nth-child(-n + 5) {
```
Comment (SanyaBratashchuk) [inline src/styles/star.css:19]:
```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
```
You can do it like that)

File: src/styles/reset.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```
Comment (SanyaBratashchuk) [inline src/styles/reset.css:11]:
```suggestion
body, p, h2 {
  margin: 0;
}

```

File: src/styles/star.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  background-image: url(../images/star-active.svg);
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```
Comment (SanyaBratashchuk) [inline src/styles/star.css:25]:
```suggestion
}

```
And change file's name to stars

File: src/styles/reset.css (hunk 3 above)
Comment (SanyaBratashchuk) [reply src/styles/reset.css:11]: It is a good practice to leave one blank line in the end of the file

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
```
Comment (SanyaBratashchuk) [inline src/style.css:43]: Move card to the separate file card.scss

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by alexandra-protyanova — APPROVED** [review]
Well done!

**Pull request closed without merging**